lib/TPP/ToBlockLayoutAndBack.cpp

```diff
@@ -337,9 +388,13 @@ packMatmulOpImpl(RewriterBase &rewriter, linalg::MatmulOp matmulOp,
   OpFoldResult tileOnI = tiles[0];
   OpFoldResult tileOnJ = tiles[1];
   OpFoldResult tileOnK = tiles[2];
+  bool isMatmulOp = std::is_same_v<OpTy, linalg::MatmulOp>;
```

Comment:
This does not make clear what else it could be.

Reply:
I have a static assert few lines above to make sure we are dealing with matmul or batch matmul. Any suggestion on how to improve?